runtime/v2/sdk/src/main/java/com/walmartlabs/concord/runtime/v2/sdk/ELFunction.java

```diff
@@ -0,0 +1,42 @@
+package com.walmartlabs.concord.runtime.v2.sdk;
+
+/*-
+ * *****
+ * Concord
+ * -----
+ * Copyright (C) 2017 - 2025 Walmart Inc.
+ * -----
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =====
+ */
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Marks a static method as a function that can be called in JUEL expressions.
+ * The method must be static and will be registered with the provided name or
+ * using the method's name if omitted.
+ * Use ":" to separate the prefix and the local name of the function.
```

Comment:
I think there should be a test that covers the `prefix:localName` support. Is there an existing one prior to this PR that I'm overlooking?